apps/api/src/routes/v1_keys_updateKey.ts

```diff
@@ -80,6 +80,27 @@ const route = createRoute({
                 "The number of requests that can be made with this key before it becomes invalid. Set `null` to disable.",
               example: 1000,
             }),
+            refill: z
+              .object({
+                interval: z.enum(["daily", "monthly"]).openapi({
+                  description:
+                    "Unkey will automatically refill verifications at the set interval. If null is used the refill functionality will be removed from the key.",
+                }),
+                amount: z.number().int().min(1).positive().openapi({
```

Comment:
```suggestion
                amount: z.number().int().min(1).openapi({
```
min(1) already ensures it's positive